dev_tools/requirements/deps/mypy.txt

```diff
@@ -1,3 +1,8 @@
 # the mypy dependency file
 mypy~=1.9
 mypy-protobuf
+
+# Newer sympy versions cause many [operator] mypy errors
+# including ones seemingly unrelated to sympy
+# https://github.com/quantumlib/Qualtran/pull/1167
+sympy<1.13
```

Comment:
Should we open an issue with sympy with a minimal example?

Reply:
on the sympy repository? I'm not convinced this isn't our fault :p

Reply:
Oh my bad, I didn't see any qualtran code in the minimal example so I thought sympy is running into errors unrelated to Qualtran. I now see the error messages were generated by Qualtran.